styles/app.scss

```diff
@@ -4,3 +4,81 @@
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import 'base';
+@import 'variables';
+
+.modal-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  background: $haze;
+ }
+
+.card {
+  width: 66%;
+  background-color: $brand-white;
+  box-shadow: $brand-shadow;
+}
+
+.card-title {
+  // flex-flow: column;
+  margin: ($gutter * 2) $gutter $gutter;
+}
+
+.form-items {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: $gutter;
+}
+
+.form-field {
+  flex-grow: 1;
+  width: 29%;
```

Comment:
set the width to 1/3 and then subtract the margins not 29%.

Also set a `min-width` of 18rem so that the grid looks right at phone sizes